packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentRecipient.test.tsx

```diff
@@ -4,6 +4,7 @@ import { paymentInfo } from '../../../__mocks__/ExternalRegistry.mock';
 import { notificationToFe, payments } from '../../../__mocks__/NotificationDetail.mock';
```

Comment:
1. Here we can add the test that checks the page from cache
2. in the test "should call handleFetchPaymentsInfo on pagination click" we can check that the page is stored in cache